lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -271,6 +283,22 @@ def write(chunk)
             entry['metadata']['severity'] = 'DEFAULT'
           end
 
+          # If a field is present in the label_map, send its value as a label
+          # (mapping the field name to label name as specified in the config)
+          # and do not send that field as part of the payload.
+          if !label_map.nil?
```

Comment:
Since the label map is constant, wouldn't it be better to put those values into "commonLabels" and just remove payload fields here?